sympy/physics/units/tests/test_dimensions.py

```diff
@@ -73,3 +84,6 @@ def test_mul_div_exp():
 
     assert length != 1
     assert length / length != 1
+
+    length_0 = length ** 0
+    assert length_0.get_dimensional_dependencies() == {}
```

Comment:
I noticed that you have removed the `expr.exp == 0` case. The first thought I had was that case, so I just wrote that. But I think that's not important at all.